src/database/migrations/20221025150522_create_fill_event_table.ts

```diff
@@ -0,0 +1,41 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.createTable('fill_event', (table) => {
+    table.increments('id');
+    table.uuid('user').references('id').inTable('user').notNullable();
+    table
+      .uuid('cylinder_set')
+      .references('id')
+      .inTable('diving_cylinder_set')
+      .notNullable();
+    table.string('info', 2048); // arbitrary number
+    table.timestamps(true, true);
+  });
+
+  await knex.schema.createTable('fill_cylinder_event', (table) => {
```

Comment:
Tän pitäisi olla varmaan 
```suggestion
  await knex.schema.createTable('fill_cylinder_set_event', (table) => {
```
Käsittääkseni ne välisillalliset tuplapullot ainakin täytetään yhtaikaa

Reply:
Sit jos käyttäjällä on tuplasetti ilman välisiltaa (en tiedä onko tälläisiä olemassa) ja niihin meneekin eri kamaa niin auta armias en tiedä mitä meidän pitäisi tehdä.

Reply:
Taulu on nimetty ihan tarkoituksella noin. Ajatuksenani oli, että jokainen pullo täytetään erikseen vaikka ne onkin aina setissä mukana. Mutta jos onkin tuommoisia atomisia tuplapulloja olemassa niin tätä ajatusta täytyy ehkä harkita uudelleen.

Reply:
Kilautin asiantuntijalle, joka kertoi että 70-luvun jälkeen tuplapullot on ollut yhteydessä toisiinsa niin että ne voi täyttää yhdestä venttiilistä.

Reply:
Ja rebeistä taasen otetaan kuulemma pullot erilleen täyttöä varten. => täytettävät "pullosetit" on siis yksittäisiä pulloja eikä meidän tarvi huolehtia siitä  koko laitepaketista tällä tasolla.